web/src/main/java/it/pagopa/selfcare/mscore/web/controller/OnboardingController.java

```diff
@@ -341,4 +341,29 @@ public ResponseEntity<byte[]> getOnboardingDocument(@ApiParam("${swagger.mscore.
         }
         return ResponseEntity.ok().headers(headers).body(file.getData());
     }
+
+    /**
+     * The function consume token onboarding request without verify method
+     *
+     * @param tokenId String
+     * @param contract MultipartFile
+     * @return no content
+     * * Code: 204, Message: successful operation, DataType: TokenId
+     * * Code: 400, Message: Invalid ID supplied, DataType: Problem
+     * * Code: 404, Message: Not found, DataType: Problem
+     */
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @ApiOperation(value = "${swagger.mscore.token.consume}", notes = "${swagger.mscore.token.consume}")
+    @PostMapping(value = "/{tokenId}/consume")
+    public ResponseEntity<Void> consumeToken(@ApiParam("${swagger.mscore.token.tokenId}")
+                                             @PathVariable(value = "tokenId") String tokenId,
+                                             @RequestPart MultipartFile contract) {
+        log.trace("consumeToken start");
```

Comment:
consider using an interceptor if you do this kind of logging for all methods https://www.baeldung.com/spring-mvc-handlerinterceptor

Reply:
I just added an analysis task for the one you wrote above

Reply:
see also https://www.baeldung.com/spring-http-logging